packages/eslint-config-nori/index.js

```diff
@@ -214,6 +214,14 @@ module.exports = {
           {
             selector: ['parameter'],
             format: ['camelCase'],
+            custom: {
+              regex: '^_[a-zA-Z][a-zA-Z0-9]*$',
+            },
+          },
+          {
+            selector: ['parameter'],
+            format: ['camelCase'],
+            modifiers: ['unused'],
             leadingUnderscore: 'allow',
```

Comment:
This change gives us better rules: all parameters must be camelCase with an optional `_` prefix, and only allows `_` when the parameter isn't used at all.

Reply:
Optionally, we can require the underscore prefix for unused parameters (vscode suggests this to me - screenshot), but I don't want to tighter rules more and risk adding more issues

<img width="525" alt="image" src="https://user-images.githubusercontent.com/7808563/213576649-afd1f6fa-d03d-472a-b2be-3f7eb7f9b1cb.png">